docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst

```diff
@@ -18,3 +18,137 @@ pneumonia as a separate entity.
 
 .. todo::
    Describe cause model
+   
+
+Data Description
+----------------
+
+.. list-table:: Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Susceptible to neonatal sepsis and other infections
+   * - I
+     - Infected
+     - Infected with neonatal sepsis and other infections
+
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - 1-prevalence_c383
+     - 
+   * - S
+     - birth prevalence
+     - 1-birth_prevalence_c383
+     - 
+   * - S
+     - excess mortality rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c383
+     - 
+   * - I
+     - birth prevalence
+     - 1-birth_prevalence_c383
```

Comment:
Typo?  I was expecting `birth_prevalence_c383` here, not one minus it

Reply:
Sorry, its a typo.. I copied from the previous row and forgot to change it. I will modify it.